lib/commands.js

```diff
@@ -1100,6 +1100,70 @@ commands.takeScreenshot = function() {
   });
 };
 
+/**
+ * getElementScreenshot(element, cb) -> cb(err)
+ *
+ * @jsonWire POST /session/:sessionId/element/:id/screenshot
+ */
+commands.getElementScreenshot = function(element) {
+  var cb = findCallback(arguments);
+  this._jsonWireCall({
+    method: 'GET'
+    , relPath: '/element/' + element + '/screenshot'
+    , cb: callbackWithData(cb, this)
+  });
+};
+
+/**
+ * saveElementScreenshot(path, cb) -> cb(err, filePath)
+ *
+ * path maybe a full file path, a directory path (finishing with /),
+ * the screenshot name, or left blank (will create a file in the system temp dir).
+ */
+commands.saveElementScreenshot = function() {
+  var _this = this;
+  var cb = findCallback(arguments);
+  var fargs = utils.varargs(arguments);
+  var _path = fargs.all[0];
+
+  function buildFilePath(_path, cb) {
+    if(!_path) { _path = tmp.tmpdir + '/'; }
+    if(_path.match(/.*\/$/)) {
+      tmp.tmpName( {template: 'screenshot-XXXXXX.png'}, function(err, fileName) {
+        if(err) { return cb(err); }
+        cb(null, path.join( _path , fileName) );
+      });
+    } else {
+      if(path.extname(_path) === '') { _path = _path + '.png'; }
+      cb(null, _path);
+    }
+  }
+
+  buildFilePath(_path, function(err, filePath) {
+    commands.getElementScreenshot.apply(_this, [function(err, base64Data) {
+      if(err) { return cb(err); }
+      require("fs").writeFile(filePath, base64Data, 'base64', function(err) {
+        if(err) { return cb(err); }
+        cb(null, filePath);
+      });
+    }]);
+  });
+};
+
+/**
+ * clickElement(element, cb) -> cb(err)
+ *
+ * @jsonWire POST /session/:sessionId/element/:id/click
+ */
+commands.clickElement = function(element) {
```

Comment:
This method already implemented and exist in your file: https://github.com/zcmgyu/wd/blob/c9cb7519e728bbbf3214165d0a1ceed5f5d0668d/lib/commands.js#L1158